geode-core/src/main/java/org/apache/geode/cache/wan/GatewayReceiver.java

```diff
@@ -92,12 +92,14 @@
   void stop();
 
   /**
-   * Destroys this <code>GatewayReceiver</code>
-   * Removes the <code>GatewayReceiverMBean</code> associated with this <code>GatewayReceiver</code>
-   * {@link #stop() stop} should be called before calling destroy
-   * Does not remove receiver from cluster configuration
+   * Destroys this <code>GatewayReceiver</code> and removes the <code>GatewayReceiverMBean</code>
```

Comment:
In theory this is more correct but I think you can commit either:
```java
and removes the {@code GatewayReceiverMBean}
```

Reply:
Oh cool, thanks for pointing that out